src/llmq/quorums_signing_shares.cpp

```diff
@@ -1133,41 +1168,28 @@ void CSigSharesManager::Cleanup()
                 assert(m);
 
                 auto& oneSigShare = m->begin()->second;
-                LogPrintf("CSigSharesManager::%s -- signing session timed out. signHash=%s, id=%s, msgHash=%s, sigShareCount=%d\n", __func__,
-                          signHash.ToString(), oneSigShare.id.ToString(), oneSigShare.msgHash.ToString(), count);
+
+                std::string strMissingMembers;
+                if (LogAcceptCategory("llmq")) {
+                    auto quorum = quorums[std::make_pair((Consensus::LLMQType)oneSigShare.llmqType, oneSigShare.quorumHash)];
```

Comment:
Could it be that it's a timed out session for an inactive quorum? Would probably be safer to use smth like
```
auto it = quorums.find(std::make_pair((Consensus::LLMQType)oneSigShare.llmqType, oneSigShare.quorumHash));
if (it != quorums.end()) {
    auto quorum = it->second;
    for(....
    ...
}
```

Reply:
Yepp, considered this as well but then decided to leave it as it is because default-construction of non-existent entries in a map would lead to `quorum == nullptr`. But you're right, better to make this more explicit, will add a commit for this.